xtask/src/commands/compliance.rs

```diff
@@ -12,7 +12,11 @@ use xtask::*;
 static LICENSES_HTML_PATH: &str = "licenses.html";
 
 #[derive(Debug, StructOpt)]
-pub struct Compliance {}
+pub struct Compliance {
+    /// Do not run the compliance test
+    #[structopt(long)]
+    skip_compliance: bool,
```

Comment:
Added this option to allow us to run tests until we have upgraded the project for new cargo about.